tests/ovs_test.go

```diff
@@ -35,4 +38,52 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		const bridgeName = "br-test"
```

Comment:
Please keep the definition in its relevant context. So inside the following Context

Reply:
Done